docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingSaveAndLoad.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Transforms;
+
+namespace Samples.Dynamic
+{
+    public static class CustomMappingSaveAndLoad
```

Comment:
>CustomMappingSaveAndLoad [](start = 24, length = 24)

Do we really need two samples? They are identical except for the serialization / deserialization stage.